experimental/packages/otlp-exporter-base/src/OTLPExporterBase.ts

```diff
@@ -110,6 +110,10 @@ export abstract class OTLPExporterBase<
     return this._shutdownOnce.call();
   }
 
+  forceFlush(): Promise<void> {
```

Comment:
ForceFlush should not resolve until all in-flight requests are completed because I want to know if the process is safe to shut down. If this can't be determined in the base class then it should be left abstract and the child classes should implement it.

Reply:
okay I wasn't confident of the implementation for these downstream exporters so I created a placeholder, should have made that clear. I will work on updating the child classes